frontend/static/content/mapping_metrics_definition.json

```diff
@@ -67,20 +89,31 @@
           "fulfilled_if": ">",
           "max_value": -1,
           "min_value": 0,
+          "description_component": "Number of user complaints per month.",
+          "example_component": "With a total of 15 user complaints per month, an entry of 15 follows.",
+          "example_process" : "In a process with 15 components and a target value of 60 complaints per month, an input of 4 follows.",
           "helper_text": "With a total of 15 user complaints per month, an entry of 15 follows."
         },
         "training": {
           "name": "Training",
           "fulfilled_if": ">",
           "max_value": 1,
           "min_value": 0,
+          "description_component": "Have all users of the component been adequately trained?",
+          "example_component": "If this is the case, an input of 1, otherwise 0, follows.",
+          "description_process": "", 
+          "example_process" : "If 12 out of 15 components have sufficient user training, an input of 80 [%] follows.",
           "helper_text": "If this is the case, an input of 1, otherwise 0, follows."
```

Comment:
Wie spielt der Helper Text da noch mit rein? Verwenden wir diesen noch oder kann der komplett rausgelöscht werden bei allen Metriken? (dann bitte auch machen)
Wie ist die Absprache wenn es kein "description_process" gibt? Ist der key dann nicht vorhanden oder ein leerer String? Aktuell ist es inkonsistent. Auf der Prozessansicht werden auch die Texte für die Komponenten angezeigt. Wie ist da die Absprache mit der fachlichen Seite/das allgemeine Verständnis? Brauchen wir überhaupt spezifische Prozess und Komponententexte für die Metriken? Wenn ja, dann bitte auch verwenden, wenn nein, bitte die nicht verwendeten Texte löschen.

Reply:
das hat aaron in seinem issue bearbeitet

Reply:
description process üebrnimmt wenn von der fachlichen gilde nicht definiert die beschreibung von component

Reply:
ISt scheinbar nicht korrekt implementiert worden ich kümmer mich drum... hab ich übersehen